code/nxsOnto/nxsOnto/generator.py

```diff
@@ -106,25 +106,31 @@
     The version string is the NeXus version followed by the ontology version.
     
 '''
-
+'''
 # To avoid re-parsing NeXus files after initial run, execute this
```

Comment:
All of this is commented out code! Not sure why you want it too as you run this script once.